### PR TITLE
Expander hide details summary

### PR DESCRIPTION
**Pull request by joe-watkins**
Due to a known bug with recent versions of iOS VoiceOver not announcing state we are encouraging use of ARIA Disclosure over Details/Summary until we get a few major versions beyond iOS 16-17.3

- Remove Details/Summary from Expander component page
- Add note about support issues w/Details element
- Convert Checklist accordions from Details/Summary to ARIA Disclosure 
- Note: Icon placement is different than Details/Summary but pretty much the same UX. A separate PR could be done to make those match exactly.

Will not be a ton of work to reintroduce Details/Summary down the road.

### CODE REVIEW

**Review by tritland — DISMISSED** [review]

File: _checklist-web/expander.md [hunk 1]
```diff
@@ -68,33 +68,32 @@ wcag:
...
 </example>
 {:/}
 
-### Use semantic HTML where possible
-This custom expander uses a semantic button with `aria-expanded` with additional scripting to toggle content and states.
+## Native expander (details/summary)
+- This semantic HTML contains all accessibility features by default with no scripting required.
+- It uses [CSS pseudo attributes](https://github.com/tmobile/magentaA11y/blob/main/_sass/modules/_details-summary.scss) to create the expanded/collapsed indicator, no Javascript.
+
+**Note:** Due to known accessibility support issues in recent versions of iOS, `<details>` is not recommended for expander accordions. We recommend the use of an ARIA Disclosure Widget which has very robust support.
```
Comment (silverli) [inline _checklist-web/expander.md:89]: nice

File: _layouts/checklist-native.html [hunk 2]
```diff
@@ -17,16 +17,20 @@ <h1 class="h-alpha">
...
-          </summary>
-          <ul>
-            {% for page in site.checklist-native %}
-            {% include checklist-loop.html %}
-            {% endfor %}
-          </ul>
-        </details>
+        <div class="checklist-details fake-details">
+          <div class="expander-group temp-details-summary-fix">
+            <button type="button" class="expander-toggle" aria-expanded="true" id="{{ category | escape }}-details" class="checklist-details">
```
Comment (silverli) [inline _layouts/checklist-native.html:22]: Great catch. I notice the existing prod version had the carat turn pink on focus FYI and this is not doing that

File: _layouts/checklist-native.html [hunk 3]
```diff
@@ -17,16 +17,20 @@ <h1 class="h-alpha">
...
+              <strong>{{ category | capitalize }}</strong>
+            </button>
+            <div class="expander-content visible" aria-hidden="false">
```
Comment (silverli) [inline _layouts/checklist-native.html:25]: aria-hidden feels unecessary here with CSS display block and none being toggled

File: _layouts/checklist-web.html [hunk 4]
```diff
@@ -18,16 +18,20 @@ <h1 class="h-alpha">
...
-            <ul>
-              {% for page in ordered_pages %}
-              {% include checklist-loop.html %}
-              {% endfor %}
-            </ul>
-          </details>
+          <div class="checklist-details fake-details">
+            <div class="expander-group temp-details-summary-fix">
+              <button type="button" class="expander-toggle" aria-expanded="true" id="{{ category | escape }}-details" class="checklist-details">
+                <strong><span aria-hidden="true">&lt;</span>{{ category | escape }}<span aria-hidden="true">&gt;</span></strong>
```
Comment (silverli) [inline _layouts/checklist-web.html:24]: `<strong>` is also unexpected to me here. Would it be better to use CSS to impart this styling?

File: _sass/modules/_details-summary.scss [hunk 5]
```diff
@@ -1,4 +1,6 @@
-details {
+// Temp: Details Summary Fix
+// .fake-details addition
```
Comment (silverli) [inline _sass/modules/_details-summary.scss:2]: lol

**Review by silverli — CHANGES_REQUESTED** [review]
Hey Joe, this is great! Thanks so much for making these changes. I had a couple suggestions. Additionally, should we keep the details/summary videos for a later use if we switch back to that pattern? Maybe rename them with an underscore? _expander.mp4a?  Also if we end up doing that, I'd leave a comment about it somewhere in the expander.md file

File: _layouts/checklist-native.html (hunk 2 above)
Comment (silverli) [reply _layouts/checklist-native.html:22]: (Not a huge deal, just noting the regression)

Comment (joe-watkins) [reply _layouts/checklist-native.html:22]: Added magenta to :hover / :focus

File: _layouts/checklist-native.html (hunk 3 above)
Comment (joe-watkins) [reply _layouts/checklist-native.html:25]: @silverli I agree but this appears to be an issue outside of the scope of this ticket if ya look at the file you called this out on `_layouts/checklist-native.html` Let's put a pin in this for now.

File: _layouts/checklist-web.html (hunk 4 above)
Comment (joe-watkins) [reply _layouts/checklist-web.html:24]: @silverli This is a change not applicable to this work and is more of a template issue. Let's put this off to the side for now :)

**Comment by joe-watkins** [comment]
@silverli Fixes are in :) I've reintroduced the backed up videos and made sure the chevron is brand magenta on hover/focus. The other callouts were more template related so skipping for now.

**Review by silverli — APPROVED** [review]
Looks great!

**Pull request merged**